src/sentry/incidents/subscription_processor.py

```diff
@@ -562,6 +622,104 @@ def process_update(self, subscription_update: QuerySubscriptionUpdate) -> None:
         # before the next one then we might alert twice.
         self.update_alert_rule_stats()
 
+    def get_anomaly_detected(
+        self, anomaly
+    ) -> bool:  # TODO: add the anomaly type once that's added to Sentry
+        """
+        Helper function to determine whether we care about an anomaly based on the
+        anomaly type and the alert rule's threshold type.
+        """
+        # need to define constants for anomaly types
+        anomaly_type = anomaly.get("anomaly").get("anomaly_type")
```

Comment:
`.get()` has a default value of `None` (like if `anomaly'` isn't found) so to safely chain them you can add a default empty dict like `anomaly_type = anomaly.get("anomaly", {}).get("anomaly_type")` otherwise you risk the 2nd `.get()` erroring because you can't do `.get()` on `None`